Bottom: 0fcfcdca675ee77db6887f913739f84c8377ad76
Top:    438e4fc39c1bf607187bca6fd6f57145ed52945f
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-28 00:50:19 +0100

Refresh of draw_funcs-attempt-at-differen

---

diff --git a/src/draw_funcs.c b/src/draw_funcs.c
index 56e0ecf..45c3c82 100644
--- a/src/draw_funcs.c
+++ b/src/draw_funcs.c
@@ -435,8 +435,8 @@ draw_layer (LayerType *layer, const BoxType *drawn_area, void *userdata)
 
   if (layer_num < max_copper_layer)
     {
-      r_search (PCB->Data->pin_tree, drawn_area, NULL, hole_callback, NULL);
-      r_search (PCB->Data->via_tree, drawn_area, NULL, hole_callback, NULL);
+      r_search (PCB->Data->pin_tree, drawn_area, NULL, pin_hole_callback, NULL);
+      r_search (PCB->Data->via_tree, drawn_area, NULL, via_hole_callback, NULL);
     }
 
   /* print the non-clearing polys */
